app/src/main/java/com/infomaniak/drive/data/models/UploadFile.kt

```diff
@@ -134,9 +142,9 @@ open class UploadFile(
             }
         }
 
-        fun notExists(uri: Uri, lastModified: Date): Boolean {
+        fun canUpload(localUri: Uri, lastModified: Date): Boolean {
             return getRealmInstance().use { realm ->
-                syncFileByUriQuery(realm, uri.toString())
+                realm.where(UploadFile::class.java).equalTo(UploadFile::originalLocalUri.name, localUri.toString())
```

Comment:
Could we use `toSafeString()` method from `Uri` object instead of `toString()` ?

Reply:
No need for that here, we don't do any logging here, no need to format this uri knowing it's just a file path

Reply:
I see, thanks for your answer 👍